api/src/test/java/bio/terra/tanagra/app/controller/UnderlaysApiControllerTest.java

```diff
@@ -24,13 +24,36 @@ public class UnderlaysApiControllerTest extends BaseSpringUnitTest {
 
   @Autowired private UnderlaysApiController controller;
 
+  // The static string variables below are only included here because they are required
```

Comment:
This should be super minimal since it's only for this one test. I would define a very short version as a constant and assert that it's passed through correctly in the `getUnderlay` test.